src/VisualStudio/Core/Next/Remote/ServiceHubRemoteHostClient.WorkspaceHost.cs

```diff
@@ -33,13 +33,6 @@ private class WorkspaceHost : ForegroundThreadAffinitizedObject, IVisualStudioWo
                 _currentSolutionId = workspace.CurrentSolution.Id;
             }
 
-            public Task InitializeAsync()
```

Comment:
Not necessary as 'StartSendingEventsToWorkspaceHost' will properly send the events that cause us to initialize ourselves with the primary workspace.